server/sonar-server/src/main/java/org/sonar/server/platform/ws/DbMigrationJsonWriter.java

```diff
@@ -37,6 +37,7 @@
   static final String STATUS_MIGRATION_SUCCEEDED = "MIGRATION_SUCCEEDED";
   static final String STATUS_MIGRATION_REQUIRED = "MIGRATION_REQUIRED";
 
+  static final String NO_CONNECTION_TO_DB = "Can not connect to Database.";
```

Comment:
@ganncamp what do you think? "Can not" or "Cannot"? I believe the second is less correct English.